src/components/FullHierarchyTable/FullHierarchyTable.test.tsx

```diff
@@ -39,6 +39,7 @@ describe('FullHierarchyTable test', () => {
         onSelect: jest.fn(),
         onDeselectAll: jest.fn(),
         onSelectAll: jest.fn(),
+        onSelectFullTable: jest.fn(),
```

Comment:
This actually tells you that it is a breaking change; consider unit test a miniature client app, if it breaks, then the client application who import our unconnected `<FullHierarchyTable />` component will also break, meaning `onSelectFullTable` property should be optional